src/index.html

```diff
@@ -7,8 +7,91 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="icon" href="./images/logo.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#">
+        <img 
+          src="./images/logo.png" 
+          alt="logo"
+        >
+      </a>
+      
+
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="apple"
```

Comment:
```suggestion
              href="#apple"
```

Reply:
fix it for all links to avoid this
![image](https://user-images.githubusercontent.com/98764641/223682804-92555255-0bd3-4046-be3f-9b27756ff87a.png)